aviatrix/resource_aviatrix_spoke_gateway.go

```diff
@@ -1500,12 +1500,12 @@ func resourceAviatrixSpokeGatewayRead(d *schema.ResourceData, meta interface{})
 	if gw.EnableBgp {
 		d.Set("learned_cidrs_approval_mode", gw.LearnedCidrsApprovalMode)
 		d.Set("bgp_polling_time", gw.BgpPollingTime)
-		d.Set("bgp_bdf_polling_time", gw.BgpBfdPollingTime)
+		d.Set("bgp_bfd_polling_time", gw.BgpBfdPollingTime)
```

Comment:
Are there tests to catch this kind of error?

Reply:
Yes I have added test to resource_avaitrix_spoke_gateway_test.go